### DISCUSSION TITLE
Project Terminate Issue

### DISCUSSION THREAD

**Discussion by risuvoo** [tailwindlabs/tailwindcss#14369] [Help]
I'm encountering an issue while working on my project using the Tailwind CLI. When I run the npm run dev command, the application runs without any problems initially. However, almost every time I make changes, I receive an "Execution Break" error, as shown in this [screenshot](https://prnt.sc/QLgpsckUA82e). To resolve this, I have to stop and rerun npm run dev repeatedly.

For context, the npm run dev command in my project is set up as follows:
```
"scripts": {
  "dev": "npx tailwindcss -i ./src/input.css -o ./src/output.css --watch"
}
```
I have already added this script to my package.json file.

This is the error: [https://prnt.sc/l4fmKXnTHj0T](https://prnt.sc/l4fmKXnTHj0T)

**Comment by wongjn** [comment]
Consider checking your `content` file globs are not overly broad.

**Reply by risuvoo** [reply]
Ok, Now what can i do?
